### ISSUE TITLE
`EnforcedStyleAlignWith: start_of_line` behaves strangely

### ISSUE THREAD

**Issue by gerrywastaken** [rubocop-hq/rubocop#6382]
See the third `good` example under the `start_of_line` section in the docs:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/EndAlignment

Copied here for posterity:

```ruby
variable =
  if true
  end
```

--------

## Expected behavior

This passes
```ruby
variable =
  if true
    :foo
  else
    :bar
  end
```

## Actual behavior

It errors with:
```
bleh.rb:5:1: C: Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.
    :foo
^^^^
bleh.rb:6:3: C: Layout/ElseAlignment: Align else with variable.
  else
  ^^^^
```

but this passes:
```ruby
variable =
  if true
  :foo
else
  :bar
  end
```

## Steps to reproduce the problem

with the following in your config:
```yaml
Layout/EndAlignment:
  EnforcedStyleAlignWith: start_of_line
```

run rubocop on a file containing this:

```ruby
variable =
  if true
    :foo
  else
    :bar
  end
```

## RuboCop version

```
$ [bundle exec] bundle exec rubocop -V
0.59.2 (using Parser 2.5.1.2, running on ruby 2.5.1 x86_64-darwin16))
```

**Comment by gerrywastaken** [comment]
I have tested the fix yet, but I just wanted to say thanks for working on it. :)
